geonode/layers/tests.py

```diff
@@ -1342,6 +1342,65 @@ def setUp(self):
         self.passwd = 'admin'
         self.anonymous_user = get_anonymous_user()
 
+    @on_ogc_backend(geoserver.BACKEND_PACKAGE)
+    @override_settings(UPLOADER=GEONODE_REST_UPLOADER)
+    def test_geonode_upload_kml_and_import_isocategory(self):
+        """
+        Ensure a KML-File can be uploaded and the category is imported correctly from accompanying ISO-XML
+        """
+
+        filename_suffix_list = [
+        'structure',        # lower case default category
+        'planningCadastre', # mixed case default category
```

Comment:
@stefmec please fix the PEP8 formatting errors:

^@^@geonode/layers/tests.py:1354:28: E261 at least two spaces before inline comment
geonode/layers/tests.py:1368:21: E128 continuation line under-indented for visual indent
geonode/layers/tests.py:1370:21: E128 continuation line under-indented for visual indent
geonode/layers/tests.py:1404:5: E303 too many blank lines (2